### PR TITLE
refactor(disk-loss): Adding the task banner in the disk-loss experiment

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

**What this PR does / why we need it**:

- Adding the task banner in the disk-loss experiment

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #1423 

**Checklist**

* [x] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [x] Have you added debug messages where necessary? 
* [x] Have you added task comments where necessary? 
* [x] Have you tested the changes for possible failure conditions?
* [x] Have you provided the positive & negative test logs for the litmusbook execution?
* [x] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [x] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [x] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [x] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [x] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [x] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [x] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [x] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ispeakc0de** [comment]
```
2020-04-13 21:30:55.465232 ************ BRACE YOURSELF, EXPERIMENT BEGINS! ************ 

2020-04-13 21:30:58.294707 Step: Gathering Facts 
task path: /experiments/generic/disk_loss/disk_loss_ansible_logic.yml:1
META: ran handlers

2020-04-13 21:30:58.650480 Step: [PreReq] Identify the chaos util for disk-loss experiment 
task path: /experiments/generic/disk_loss/disk_loss_ansible_prerequisites.yml:1
details => {"changed": true, "checksum": "4c0f65555707eef0a34ca39e7295f9f9a8330975", "dest": "./chaosutil.yml", "gid": 0, "group": "root", "md5sum": "1ba40dc0fa321facce28f77bf6107988", "mode": "0644", "owner": "root", "size": 155, "src": "/root/.ansible/tmp/ansible-tmp-1586813458.34-184034342703287/source", "state": "file", "uid": 0}

2020-04-13 21:30:58.696157 Step: [PreReq]: Including the chaos util for the disk-loss experiment 
task path: /experiments/generic/disk_loss/disk_loss_ansible_logic.yml:28
details => {"ansible_facts": {"c_util": "/chaoslib/litmus/platform/gke/disk_loss.yml"}, "ansible_included_var_files": ["/experiments/generic/disk_loss/chaosutil.yml"], "changed": false}

2020-04-13 21:30:58.744513 Step: [PreReq]: Constructing the chaos result name 
task path: /experiments/generic/disk_loss/disk_loss_ansible_logic.yml:33
details => {"ansible_facts": {"c_result": "engine1-disk-loss"}, "changed": false}
included: /utils/runtime/update_chaos_result_resource.yml for 127.0.0.1

2020-04-13 21:30:59.006703 Step: [PreReq]: Generate the chaos result CR to reflect SOT (Start of Test) 
task path: /utils/runtime/update_chaos_result_resource.yml:3
details => {"changed": true, "checksum": "c115f41ece5300bae6d747ceb7ff2dff91f768e1", "dest": "/utils/runtime/chaos-result.yml", "gid": 0, "group": "root", "md5sum": "73fcf7cde93a40d81dd3104bf460ff0b", "mode": "0644", "owner": "root", "size": 304, "src": "/root/.ansible/tmp/ansible-tmp-1586813458.84-35083277971788/source", "state": "file", "uid": 0}

2020-04-13 21:31:00.007796 Step: [PreReq]: Apply the chaos result CR for disk-loss experiment 
task path: /utils/runtime/update_chaos_result_resource.yml:13
details => {"changed": true, "cmd": "kubectl apply -f /utils/runtime/chaos-result.yml -n shubham", "delta": "0:00:00.763431", "end": "2020-04-13 21:30:59.985358", "failed_when_result": false, "rc": 0, "start": "2020-04-13 21:30:59.221927", "stderr": "", "stderr_lines": [], "stdout": "chaosresult.litmuschaos.io/engine1-disk-loss configured", "stdout_lines": ["chaosresult.litmuschaos.io/engine1-disk-loss configured"]}

2020-04-13 21:31:00.108690 Step: [Info]: Display the application information passed via the test job 
task path: /experiments/generic/disk_loss/disk_loss_ansible_logic.yml:46
details => {
    "msg": [
        "The application info is as follows:", 
        "Namespace    : shubham", 
        "Label        : run=nginx", 
        "Ramp Time    : "
    ]
}
included: /utils/common/status_app_pod.yml for 127.0.0.1

2020-04-13 21:31:00.686038 Step: [Status]: Checking whether application pods are in running state 
task path: /utils/common/status_app_pod.yml:2
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pods -n shubham -l run=nginx -o custom-columns=:.status.phase --no-headers", "delta": "0:00:00.385124", "end": "2020-04-13 21:31:00.664916", "rc": 0, "start": "2020-04-13 21:31:00.279792", "stderr": "", "stderr_lines": [], "stdout": "Running", "stdout_lines": ["Running"]}

2020-04-13 21:31:01.211764 Step: [Status]: Checking whether application containers are in running state 
task path: /utils/common/status_app_pod.yml:9
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pod -n shubham -l run=nginx --no-headers -o jsonpath='{.items[*].status.containerStatuses[*].ready}' | tr ' ' '\\n' | uniq", "delta": "0:00:00.375940", "end": "2020-04-13 21:31:01.192188", "rc": 0, "start": "2020-04-13 21:31:00.816248", "stderr": "", "stderr_lines": [], "stdout": "true", "stdout_lines": ["true"]}
included: /utils/common/generate-kubernetes-chaos-events.yml for 127.0.0.1

2020-04-13 21:31:01.762313 Step: [Event]: Obtain the current date in ISO8601 format 
task path: /utils/common/generate-kubernetes-chaos-events.yml:1
details => {"changed": true, "cmd": "date --iso-8601=seconds", "delta": "0:00:00.296780", "end": "2020-04-13 21:31:01.744986", "rc": 0, "start": "2020-04-13 21:31:01.448206", "stderr": "", "stderr_lines": [], "stdout": "2020-04-13T21:31:01+00:00", "stdout_lines": ["2020-04-13T21:31:01+00:00"]}

2020-04-13 21:31:01.811627 Step: set_fact 
task path: /utils/common/generate-kubernetes-chaos-events.yml:5
details => {"ansible_facts": {"ts": "2020-04-13T21:31:01Z"}, "changed": false}

2020-04-13 21:31:02.492408 Step: [Event]: Creating a PreChaosCheck event in the chaosengine 
task path: /utils/common/generate-kubernetes-chaos-events.yml:8
details => {"changed": true, "method": "patch", "result": {"apiVersion": "v1", "count": 1, "eventTime": null, "firstTimestamp": "2020-04-13T21:31:01Z", "involvedObject": {"apiVersion": "litmuschaos.io/v1alpha1", "kind": "ChaosEngine", "name": "engine1", "namespace": "shubham", "uid": "0802fd59-7dce-11ea-afea-42010a80014c"}, "kind": "Event", "lastTimestamp": "2020-04-13T21:31:01Z", "message": "AUT is Running successfully", "metadata": {"creationTimestamp": "2020-04-13T21:06:21Z", "name": "event-\"PreChaosCheck\"", "namespace": "shubham", "resourceVersion": "488", "selfLink": "/api/v1/namespaces/shubham/events/event-%22PreChaosCheck%22", "uid": "a04b4513-7dca-11ea-afea-42010a80014c"}, "reason": "PreChaosCheck", "reportingComponent": "", "reportingInstance": "", "source": {"component": "disk-loss-tjsygo-djkn6"}, "type": "Normal"}}
included: /utils/cloud/gcp/gcloud_configure.yml for 127.0.0.1

2020-04-13 21:31:03.547209 Step: [Authentication]: authenticate gcloud service account 
task path: /utils/cloud/gcp/gcloud_configure.yml:1
details => {"changed": true, "cmd": "gcloud auth activate-service-account --key-file=/tmp/cloud_config.yml", "delta": "0:00:00.881840", "end": "2020-04-13 21:31:03.531954", "rc": 0, "start": "2020-04-13 21:31:02.650114", "stderr": "Activated service account credentials for: [karthik-test@strong-eon-153112.iam.gserviceaccount.com]", "stderr_lines": ["Activated service account credentials for: [karthik-test@strong-eon-153112.iam.gserviceaccount.com]"], "stdout": "", "stdout_lines": []}

2020-04-13 21:31:04.423774 Step: [Authentication]: Setting the project ID in the gcloud 
task path: /utils/cloud/gcp/gcloud_configure.yml:4
details => {"changed": true, "cmd": "gcloud config set project strong-eon-153112", "delta": "0:00:00.744041", "end": "2020-04-13 21:31:04.407593", "rc": 0, "start": "2020-04-13 21:31:03.663552", "stderr": "Updated property [core/project].", "stderr_lines": ["Updated property [core/project]."], "stdout": "", "stdout_lines": []}

2020-04-13 21:31:05.194905 Step: [Authentication]: Checking gcloud config list 
task path: /utils/cloud/gcp/gcloud_configure.yml:7
details => {"changed": true, "cmd": "gcloud config list", "delta": "0:00:00.630580", "end": "2020-04-13 21:31:05.178052", "rc": 0, "start": "2020-04-13 21:31:04.547472", "stderr": "\nYour active configuration is: [default]", "stderr_lines": ["", "Your active configuration is: [default]"], "stdout": "[core]\naccount = karthik-test@strong-eon-153112.iam.gserviceaccount.com\ndisable_usage_reporting = True\nproject = strong-eon-153112", "stdout_lines": ["[core]", "account = karthik-test@strong-eon-153112.iam.gserviceaccount.com", "disable_usage_reporting = True", "project = strong-eon-153112"]}
included: /utils/cloud/gcp/status_disk.yml for 127.0.0.1

2020-04-13 21:31:06.389585 Step: [Status]: Getting the disk users 
task path: /utils/cloud/gcp/status_disk.yml:1
details => {"changed": true, "cmd": "gcloud compute disks describe shubham-disk3 --zone us-central1-c --format=\"csv(users)\"", "delta": "0:00:00.971842", "end": "2020-04-13 21:31:06.371692", "rc": 0, "start": "2020-04-13 21:31:05.399850", "stderr": "", "stderr_lines": [], "stdout": "users\nhttps://www.googleapis.com/compute/v1/projects/strong-eon-153112/zones/us-central1-c/instances/gke-shubham-test-default-pool-8edc1198-z10j", "stdout_lines": ["users", "https://www.googleapis.com/compute/v1/projects/strong-eon-153112/zones/us-central1-c/instances/gke-shubham-test-default-pool-8edc1198-z10j"]}

2020-04-13 21:31:06.617595 Step: [Status]: Disk status check 
task path: /utils/cloud/gcp/status_disk.yml:5
details => {"changed": true, "checksum": "2064d406d5601347db02fe7e454062222e7fd1d4", "dest": "./disk_status_check.yml", "gid": 0, "group": "root", "md5sum": "06fa36b9c3720e6f2d07cbd493ebf8b0", "mode": "0644", "owner": "root", "size": 308, "src": "/root/.ansible/tmp/ansible-tmp-1586813466.43-217273873742397/source", "state": "file", "uid": 0}

2020-04-13 21:31:06.661485 Step: include_vars 
task path: /utils/cloud/gcp/status_disk.yml:10
details => {"ansible_facts": {"inuse": true}, "ansible_included_var_files": ["/experiments/generic/disk_loss/disk_status_check.yml"], "changed": false}

2020-04-13 21:31:06.734045 Step: debug 
task path: /experiments/generic/disk_loss/disk_loss_ansible_logic.yml:113
details => {
    "msg": "specified disk is attached to node"
}
included: /chaoslib/litmus/platform/gke/disk_loss.yml for 127.0.0.1
included: /utils/common/generate-kubernetes-chaos-events.yml for 127.0.0.1

2020-04-13 21:31:07.379085 Step: [Event]: Obtain the current date in ISO8601 format 
task path: /utils/common/generate-kubernetes-chaos-events.yml:1
details => {"changed": true, "cmd": "date --iso-8601=seconds", "delta": "0:00:00.305500", "end": "2020-04-13 21:31:07.361584", "rc": 0, "start": "2020-04-13 21:31:07.056084", "stderr": "", "stderr_lines": [], "stdout": "2020-04-13T21:31:07+00:00", "stdout_lines": ["2020-04-13T21:31:07+00:00"]}

2020-04-13 21:31:07.424255 Step: set_fact 
task path: /utils/common/generate-kubernetes-chaos-events.yml:5
details => {"ansible_facts": {"ts": "2020-04-13T21:31:07Z"}, "changed": false}

2020-04-13 21:31:07.937017 Step: [Event]: Creating a ChaosInject event in the chaosengine 
task path: /utils/common/generate-kubernetes-chaos-events.yml:8
details => {"changed": true, "method": "patch", "result": {"apiVersion": "v1", "count": 1, "eventTime": null, "firstTimestamp": "2020-04-13T21:31:07Z", "involvedObject": {"apiVersion": "litmuschaos.io/v1alpha1", "kind": "ChaosEngine", "name": "engine1", "namespace": "shubham", "uid": "0802fd59-7dce-11ea-afea-42010a80014c"}, "kind": "Event", "lastTimestamp": "2020-04-13T21:31:07Z", "message": "Injecting disk-loss chaos on shubham-disk3 disk", "metadata": {"creationTimestamp": "2020-04-13T21:06:22Z", "name": "event-\"ChaosInject\"", "namespace": "shubham", "resourceVersion": "489", "selfLink": "/api/v1/namespaces/shubham/events/event-%22ChaosInject%22", "uid": "a114681c-7dca-11ea-afea-42010a80014c"}, "reason": "ChaosInject", "reportingComponent": "", "reportingInstance": "", "source": {"component": "disk-loss-tjsygo-djkn6"}, "type": "Normal"}}

2020-04-13 21:31:13.972558 Step: [Inject]: Detaching the shubham-disk3 disk from gke-shubham-test-default-pool-8edc1198-z10j node 
task path: /chaoslib/litmus/platform/gke/disk_loss.yml:21
details => {"changed": true, "cmd": "gcloud compute instances detach-disk gke-shubham-test-default-pool-8edc1198-z10j --disk shubham-disk3 --zone us-central1-c", "delta": "0:00:05.888280", "end": "2020-04-13 21:31:13.949729", "rc": 0, "start": "2020-04-13 21:31:08.061449", "stderr": "Updated [https://www.googleapis.com/compute/v1/projects/strong-eon-153112/zones/us-central1-c/instances/gke-shubham-test-default-pool-8edc1198-z10j].", "stderr_lines": ["Updated [https://www.googleapis.com/compute/v1/projects/strong-eon-153112/zones/us-central1-c/instances/gke-shubham-test-default-pool-8edc1198-z10j]."], "stdout": "", "stdout_lines": []}

2020-04-13 21:31:29.227277 Step: [Wait]: Waiting for 15s after chaos injection 
task path: /chaoslib/litmus/platform/gke/disk_loss.yml:24
details => {"changed": false, "elapsed": 15, "path": null, "port": null, "search_regex": null, "state": "started"}
included: /utils/cloud/gcp/status_disk.yml for 127.0.0.1

2020-04-13 21:31:30.440385 Step: [Status]: Getting the disk users 
task path: /utils/cloud/gcp/status_disk.yml:1
details => {"changed": true, "cmd": "gcloud compute disks describe shubham-disk3 --zone us-central1-c --format=\"csv(users)\"", "delta": "0:00:01.042293", "end": "2020-04-13 21:31:30.424002", "rc": 0, "start": "2020-04-13 21:31:29.381709", "stderr": "", "stderr_lines": [], "stdout": "users", "stdout_lines": ["users"]}

2020-04-13 21:31:31.024061 Step: [Status]: Disk status check 
task path: /utils/cloud/gcp/status_disk.yml:5
details => {"changed": true, "checksum": "4b45fadf3b4916b824361b4ce1d60d854009b66d", "dest": "./disk_status_check.yml", "gid": 0, "group": "root", "md5sum": "edeb02117f80ce214b81c5676b2c8031", "mode": "0644", "owner": "root", "size": 256, "src": "/root/.ansible/tmp/ansible-tmp-1586813490.53-16122884958902/source", "state": "file", "uid": 0}

2020-04-13 21:31:31.069105 Step: include_vars 
task path: /utils/cloud/gcp/status_disk.yml:10
details => {"ansible_facts": {"inuse": false}, "ansible_included_var_files": ["/experiments/generic/disk_loss/disk_status_check.yml"], "changed": false}

2020-04-13 21:31:36.909695 Step: [Recover]: If disk is not attached, it will attach manually 
task path: /chaoslib/litmus/platform/gke/disk_loss.yml:32
details => {"changed": true, "cmd": "gcloud compute instances attach-disk gke-shubham-test-default-pool-8edc1198-z10j --device-name shubham-disk3 --disk shubham-disk3 --zone us-central1-c", "delta": "0:00:05.698268", "end": "2020-04-13 21:31:36.893754", "rc": 0, "start": "2020-04-13 21:31:31.195486", "stderr": "Updated [https://www.googleapis.com/compute/v1/projects/strong-eon-153112/zones/us-central1-c/instances/gke-shubham-test-default-pool-8edc1198-z10j].", "stderr_lines": ["Updated [https://www.googleapis.com/compute/v1/projects/strong-eon-153112/zones/us-central1-c/instances/gke-shubham-test-default-pool-8edc1198-z10j]."], "stdout": "", "stdout_lines": []}
included: /utils/common/status_app_pod.yml for 127.0.0.1

2020-04-13 21:31:37.551279 Step: [Status]: Checking whether application pods are in running state 
task path: /utils/common/status_app_pod.yml:2
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pods -n shubham -l run=nginx -o custom-columns=:.status.phase --no-headers", "delta": "0:00:00.392923", "end": "2020-04-13 21:31:37.530422", "rc": 0, "start": "2020-04-13 21:31:37.137499", "stderr": "", "stderr_lines": [], "stdout": "Running", "stdout_lines": ["Running"]}

2020-04-13 21:31:38.051735 Step: [Status]: Checking whether application containers are in running state 
task path: /utils/common/status_app_pod.yml:9
details => {"attempts": 1, "changed": true, "cmd": "kubectl get pod -n shubham -l run=nginx --no-headers -o jsonpath='{.items[*].status.containerStatuses[*].ready}' | tr ' ' '\\n' | uniq", "delta": "0:00:00.365363", "end": "2020-04-13 21:31:38.030878", "rc": 0, "start": "2020-04-13 21:31:37.665515", "stderr": "", "stderr_lines": [], "stdout": "true", "stdout_lines": ["true"]}
included: /utils/common/generate-kubernetes-chaos-events.yml for 127.0.0.1

2020-04-13 21:31:38.581295 Step: [Event]: Obtain the current date in ISO8601 format 
task path: /utils/common/generate-kubernetes-chaos-events.yml:1
details => {"changed": true, "cmd": "date --iso-8601=seconds", "delta": "0:00:00.306633", "end": "2020-04-13 21:31:38.564749", "rc": 0, "start": "2020-04-13 21:31:38.258116", "stderr": "", "stderr_lines": [], "stdout": "2020-04-13T21:31:38+00:00", "stdout_lines": ["2020-04-13T21:31:38+00:00"]}

2020-04-13 21:31:38.625862 Step: set_fact 
task path: /utils/common/generate-kubernetes-chaos-events.yml:5
details => {"ansible_facts": {"ts": "2020-04-13T21:31:38Z"}, "changed": false}

2020-04-13 21:31:39.135698 Step: [Event]: Creating a PostChaosCheck event in the chaosengine 
task path: /utils/common/generate-kubernetes-chaos-events.yml:8
details => {"changed": true, "method": "create", "result": {"apiVersion": "v1", "count": 1, "eventTime": null, "firstTimestamp": "2020-04-13T21:31:38Z", "involvedObject": {"apiVersion": "litmuschaos.io/v1alpha1", "kind": "ChaosEngine", "name": "engine1", "namespace": "shubham", "uid": "0802fd59-7dce-11ea-afea-42010a80014c"}, "kind": "Event", "lastTimestamp": "2020-04-13T21:31:38Z", "message": "Disk has been reattached and AUT is Running successfully", "metadata": {"creationTimestamp": "2020-04-13T21:31:39Z", "name": "event-\"PostChaosCheck\"", "namespace": "shubham", "resourceVersion": "490", "selfLink": "/api/v1/namespaces/shubham/events/event-%22PostChaosCheck%22", "uid": "28e26bb5-7dce-11ea-afea-42010a80014c"}, "reason": "PostChaosCheck", "reportingComponent": "", "reportingInstance": "", "source": {"component": "disk-loss-tjsygo-djkn6"}, "type": "Normal"}}
included: /utils/cloud/gcp/status_disk.yml for 127.0.0.1

2020-04-13 21:31:40.366862 Step: [Status]: Getting the disk users 
task path: /utils/cloud/gcp/status_disk.yml:1
details => {"changed": true, "cmd": "gcloud compute disks describe shubham-disk3 --zone us-central1-c --format=\"csv(users)\"", "delta": "0:00:01.033946", "end": "2020-04-13 21:31:40.349788", "rc": 0, "start": "2020-04-13 21:31:39.315842", "stderr": "", "stderr_lines": [], "stdout": "users\nhttps://www.googleapis.com/compute/v1/projects/strong-eon-153112/zones/us-central1-c/instances/gke-shubham-test-default-pool-8edc1198-z10j", "stdout_lines": ["users", "https://www.googleapis.com/compute/v1/projects/strong-eon-153112/zones/us-central1-c/instances/gke-shubham-test-default-pool-8edc1198-z10j"]}

2020-04-13 21:31:40.882170 Step: [Status]: Disk status check 
task path: /utils/cloud/gcp/status_disk.yml:5
details => {"changed": true, "checksum": "2064d406d5601347db02fe7e454062222e7fd1d4", "dest": "./disk_status_check.yml", "gid": 0, "group": "root", "md5sum": "06fa36b9c3720e6f2d07cbd493ebf8b0", "mode": "0644", "owner": "root", "size": 308, "src": "/root/.ansible/tmp/ansible-tmp-1586813500.41-161838579211889/source", "state": "file", "uid": 0}

2020-04-13 21:31:40.924120 Step: include_vars 
task path: /utils/cloud/gcp/status_disk.yml:10
details => {"ansible_facts": {"inuse": true}, "ansible_included_var_files": ["/experiments/generic/disk_loss/disk_status_check.yml"], "changed": false}

2020-04-13 21:31:40.997035 Step: debug 
task path: /experiments/generic/disk_loss/disk_loss_ansible_logic.yml:170
details => {
    "msg": "specified disk is attached to node"
}

2020-04-13 21:31:41.075775 Step: set_fact 
task path: /experiments/generic/disk_loss/disk_loss_ansible_logic.yml:178
details => {"ansible_facts": {"flag": "Pass"}, "changed": false}

2020-04-13 21:31:41.137701 Step: [Result]: Getting the final result of disk-loss experiment 
task path: /experiments/generic/disk_loss/disk_loss_ansible_logic.yml:181
details => {
    "msg": "disk-loss experiment has been Passed"
}
included: /utils/runtime/getting_failure_step.yml for 127.0.0.1

2020-04-13 21:31:41.421958 Step: [Failure-Detection]: Recording the name of failed step 
task path: /utils/runtime/getting_failure_step.yml:44
details => {"ansible_facts": {"failStep": "N/A"}, "changed": false}
included: /utils/common/generate-kubernetes-chaos-events.yml for 127.0.0.1

2020-04-13 21:31:41.895606 Step: [Event]: Obtain the current date in ISO8601 format 
task path: /utils/common/generate-kubernetes-chaos-events.yml:1
details => {"changed": true, "cmd": "date --iso-8601=seconds", "delta": "0:00:00.292523", "end": "2020-04-13 21:31:41.876547", "rc": 0, "start": "2020-04-13 21:31:41.584024", "stderr": "", "stderr_lines": [], "stdout": "2020-04-13T21:31:41+00:00", "stdout_lines": ["2020-04-13T21:31:41+00:00"]}

2020-04-13 21:31:41.945797 Step: set_fact 
task path: /utils/common/generate-kubernetes-chaos-events.yml:5
details => {"ansible_facts": {"ts": "2020-04-13T21:31:41Z"}, "changed": false}

2020-04-13 21:31:42.488596 Step: [Event]: Creating a Summary event in the chaosengine 
task path: /utils/common/generate-kubernetes-chaos-events.yml:8
details => {"changed": true, "method": "patch", "result": {"apiVersion": "v1", "count": 1, "eventTime": null, "firstTimestamp": "2020-04-13T21:31:41Z", "involvedObject": {"apiVersion": "litmuschaos.io/v1alpha1", "kind": "ChaosEngine", "name": "engine1", "namespace": "shubham", "uid": "0802fd59-7dce-11ea-afea-42010a80014c"}, "kind": "Event", "lastTimestamp": "2020-04-13T21:31:41Z", "message": "disk-loss Experiment Passed!", "metadata": {"creationTimestamp": "2020-04-13T21:06:45Z", "name": "event-\"Summary\"", "namespace": "shubham", "resourceVersion": "491", "selfLink": "/api/v1/namespaces/shubham/events/event-%22Summary%22", "uid": "aed2dad5-7dca-11ea-afea-42010a80014c"}, "reason": "Summary", "reportingComponent": "", "reportingInstance": "", "source": {"component": "disk-loss-tjsygo-djkn6"}, "type": "Normal"}}
included: /utils/runtime/update_chaos_result_resource.yml for 127.0.0.1

2020-04-13 21:31:43.132783 Step: [Result]: Update the chaos result CR to reflect EOT (End of Test) 
task path: /utils/runtime/update_chaos_result_resource.yml:23
details => {"changed": true, "checksum": "0a16ae530fb5ab955d97e6460112b5c89d103c2e", "dest": "/utils/runtime/chaos-result.yml", "gid": 0, "group": "root", "md5sum": "b9bdb758684884ef9c0d62bb094c7e5d", "mode": "0644", "owner": "root", "size": 321, "src": "/root/.ansible/tmp/ansible-tmp-1586813502.66-68312132067833/source", "state": "file", "uid": 0}

2020-04-13 21:31:43.794610 Step: [The END]: Apply the chaos result CR for disk-loss experiment 
task path: /utils/runtime/update_chaos_result_resource.yml:33
details => {"changed": true, "cmd": "kubectl apply -f /utils/runtime/chaos-result.yml -n shubham", "delta": "0:00:00.520462", "end": "2020-04-13 21:31:43.775549", "failed_when_result": false, "rc": 0, "start": "2020-04-13 21:31:43.255087", "stderr": "", "stderr_lines": [], "stdout": "chaosresult.litmuschaos.io/engine1-disk-loss configured", "stdout_lines": ["chaosresult.litmuschaos.io/engine1-disk-loss configured"]}
META: ran handlers
META: ran handlers

2020-04-13 21:31:43.797259 ************ RELAX, EXPERIMENT ENDS! ************ 
127.0.0.1                  : ok=59   changed=28   unreachable=0    failed=0  
```

File: chaoslib/litmus/platform/aws/disk_loss.yml [hunk 1]
```diff
@@ -1,38 +1,51 @@
...
 
-- name: Detaching the disk
+- name: "[Inject]: Detaching the {{ disk_name }} disk from {{ node_name }} node"
   ec2_vol:
     id: "{{ disk_name }}"
     instance: None
     region: "{{ zone_name }}"
 
-- name: chaos injection for {{ c_duration }}s
+- name: "[Wait]: Waiting for {{ c_duration }}s after chaos injection"
```
Comment (uditgaurav) [inline chaoslib/litmus/platform/aws/disk_loss.yml:27]: `"[Wait]: Waiting for chaos duration of {{ c_duration }}s after chaos injection"`

File: chaoslib/litmus/platform/aws/disk_loss.yml [hunk 2]
```diff
@@ -1,38 +1,51 @@
...
   wait_for:
     timeout: "{{ c_duration }}"
 
-- name: Verify that the disk is connected to node (post)
+- name: "[Status]: Verify that the disk is connected to node (post)"
   include_tasks: "/utils/cloud/aws/status_disk.yml"
   
 - block:
-    - name: If disk is not attached, it will attach manually
+    - name: "[Recover]: If disk is not attached, it will attach manually"
```
Comment (uditgaurav) [inline chaoslib/litmus/platform/aws/disk_loss.yml:35]: `If the disk is not attached ...`

File: chaoslib/litmus/platform/gke/disk_loss.yml [hunk 3]
```diff
@@ -1,31 +1,44 @@
...
+        msg: "***** Waiting for the ramp interval of {{ ramp_time }}s *****"
+
+    - name: "[Ramp]: Waiting for the specified ramp time before injecting chaos"
+      wait_for: timeout="{{ ramp_time }}"
+
+  when: "ramp_time is defined and ramp_time != ''" 
 
 ## RECORD EVENT FOR CHAOS INJECTION
-- include_tasks: /utils/common/generate-kubernetes-chaos-events.yml
+- name: "[Event]: Generating an Event for ChaosInjection"
```
Comment (uditgaurav) [inline chaoslib/litmus/platform/gke/disk_loss.yml:12]: `Generating Events for ChaosInjection"`

File: chaoslib/litmus/platform/gke/disk_loss.yml [hunk 4]
```diff
@@ -1,31 +1,44 @@
...
     engine_ns: "{{ c_ns }}"
     message: "Injecting {{ c_experiment }} chaos on {{ disk_name }} disk"
   when: "c_engine is defined and c_engine != ''"
 
-- name: Detaching the disk
+- name: "[Inject]: Detaching the {{ disk_name }} disk from {{ node_name }} node"
   shell: gcloud compute instances detach-disk {{ node_name }} --disk {{ disk_name }} --zone {{ zone_name }}
 
-- name: chaos injection for {{ c_duration }}s
+- name: "[Wait]: Waiting for {{ c_duration }}s after chaos injection"
```
Comment (uditgaurav) [inline chaoslib/litmus/platform/gke/disk_loss.yml:24]: `"[Wait]: Waiting for chaos duration of {{ c_duration }}s after chaos injection"`

**Pull request merged**